public_html/application/classes/Controller/Affiliate.php

```diff
@@ -326,7 +325,9 @@ public function _checkoldpassword (Validation $array, $field)
   public function _unique_username(Validation $array, $field)
   {
      // controllo il db
-     $name_exists = (bool) ORM::factory('user')->where('username', $array[$field])->count_all();
```

Comment:
This needs an equals

Reply:
Where so?

Reply:
`where('username', $array[$field])`
should be
`where('username', '=', $array[$field])`